yoda/llava_data_prep/config.yaml

```diff
@@ -0,0 +1,13 @@
+llm: 
+    "api": "sambastudio"
+    "do_sample": True
+    "temperature": 0.7
+    "streaming": False
+    "max_tokens_to_generate": 1200
+    "coe": True #set as true if using Sambastudio CoE endpoint
+    "select_expert": "Meta-Llama-3.1-70B-Instruct" #set if using sncloud, SambaStudio CoE llm expert
```

Comment:
Let's please remove the if using sncloud here, as this is a SambaStudio only kit as far as I'm aware.